pkg/wasm/httpfetcher.go

```diff
@@ -23,39 +25,45 @@ import (
 	"github.com/cenkalti/backoff/v4"
 )
 
+var (
+	httpInitialBackoff = time.Millisecond * 500
```

Comment:
any particular reason to move this out, instead of keeping it as part of `HTTPFetcher`?

Reply:
It is just a sort of constant rather than instance-wise variable, so I refactored it.